design/authn_azure/authn_azure_solution_design.md

```diff
@@ -420,44 +420,44 @@ Azure authenticator performance should conform with our other authenticators wit
+| 6  | No Conjur Host identity exists for Azure VM                                                                                 | VM sends authentication request with a valid Azure token            | Return 401 Unauthorized error and `RoleNotFound` message appears in log                  | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 7  | authn-azure is/is not whitelisted in ENV or enabled in DB (3 separate tests)                                                | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `AuthenticatorNotEnabled` message appears in log       | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 8  | Policy is loaded w/o `provider-uri` variable                                                                                | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `RequiredResourceMissing` message appears in log       | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 9  | Policy is loaded with `provider-uri` variable but a value is never added                                                    | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `RequiredSecretMissing` message appears in log         | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 10 | Policy is loaded w/o defining authenticator webservice                                                                      | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `WebserviceNotFound` message appears in log            | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 11 | An invalid/non-existent `provider-uri` (Conjur couldn't connect to provider)                                                | VM sends authentication request to Conjur with valid Azure token    | Return 504 GatewayTimeout error and `ProviderDiscoveryTimeout` message appears in log    | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 12 | Conjur can make contact with provider but couldn't confirm token signature                                                  | VM sends authentication request to Conjur with invalid Azure token  | Return 502 Bad Gateway error and `ProviderTokenInvalid` message appears in log           | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 13 | One or more annotations combination is incorrect or missing from the host policy (6 separate tests, see below explanation)  | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `RoleMissingAnnotations` message appears in log        | Integration               | <ul><li> [ ] Tested</li></ul>| partially |
```

Comment:
Can you add a comment on `partially` stating that more info is shown in the table below?

Reply:
Done